pkg/tenant/organizations.go

```diff
@@ -61,17 +64,45 @@ func (s *Server) OrganizationDetail(c *gin.Context) {
 		return
 	}
 
-	// Build the response from the Quarter
+	// Build the response from the Quarterdeck response
 	out := &api.Organization{
 		ID:       org.ID.String(),
 		Name:     org.Name,
 		Domain:   org.Domain,
 		Created:  org.Created.Format(time.RFC3339Nano),
 		Modified: org.Modified.Format(time.RFC3339Nano),
 	}
+
+	// Get the organization owner
+	if out.Owner, err = getOwner(ctx, org); err != nil {
+		log.Error().Err(err).Str("org", org.ID.String()).Msg("could not retrieve organization owner")
+	}
```

Comment:
An organization not having an owner should probably not happen so we want to know about it. However, I don't think returning an error to the user is necessary (it just blocks the user from being able to view the rest of the organization record). Let me know what you think about that.

Reply:
agreed